modules/ts/include/opencv2/ts.hpp

```diff
@@ -117,7 +117,9 @@
 #endif
 
 #if defined(__OPENCV_BUILD) && defined(__clang__)
+#pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Winconsistent-missing-override"
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
```

Comment:
Need to merge it with line 128.

There are 1400+ warnings for clang 6.0: http://pullrequest.opencv.org/buildbot/builders/precommit_custom_linux/builds/100200

Reply:
Done, there are only 11 warnings now: 10 in ADE (should be fixed by opencv/ade#33) and 1 in sample.

Reply:
I have not checked clang-6 though, will try it later.